test/test-utils.ts

```diff
@@ -231,7 +245,11 @@ function mockApis (options: MockApiOptions = {}): MockApiReturns {
         if (ws.__authSent) {
           send(`<stream:features xmlns:stream="http://etherx.jabber.org/streams"><bind xmlns="urn:ietf:params:xml:ns:xmpp-bind"></bind><session xmlns="urn:ietf:params:xml:ns:xmpp-session"></session></stream:features>`);
         } else {
-          send('<stream:features xmlns:stream="http://etherx.jabber.org/streams"><mechanisms xmlns="urn:ietf:params:xml:ns:xmpp-sasl"><mechanism>PLAIN</mechanism></mechanisms></stream:features>');
+          if (guestSdk) {
+            send('<stream:features xmlns:stream="http://etherx.jabber.org/streams"><mechanisms xmlns="urn:ietf:params:xml:ns:xmpp-sasl"><mechanism>ANONYMOUS</mechanism></mechanisms></stream:features>');
```

Comment:
@djhouseknecht this was the primary thing that was preventing guests from initializing in the tests